configuration/outputs.md

```diff
@@ -146,6 +152,7 @@ If enabled, the Scalyr integration will stream analytics and insights to your Sc
 | name | User defined name of this specific destination, used for mapping this destination to a workflow | Yes |
 | type | Streaming destination type \(i.e. scalyr, influxdb, sumologic, datadog, etc.\) | Yes |
 | endpoint | Scalyr endpoint | Yes |
+| features | Features defines which data types stream to backend. Only "log" is supported. | No |
```

Comment:
I thought Scalyr supported both logs + metrics, but the metrics come in via the logs (uploaddata) api?

Reply:
it is corrected, thanks